cfme/tests/cloud/test_keypairs.py

```diff
@@ -190,3 +186,25 @@ def test_download_private_key(keypair):
         initialEstimate: 1/4h
     """
     keypair.download_private_key()
+
+
+@pytest.mark.meta(automates=[1741635, 1747179])
+@test_requirements.multi_tenancy
+def test_view_keypair(appliance, new_tenant_admin):
+    """
+    Test to verify that child tenants can see key pairs of parent tenants.
+
+    Polarion:
+        assignee: nachandr
+        casecomponent: Configuration
+        caseimportance: high
+        tags: cfme_tenancy
+        initialEstimate: 1/4h
+    """
+    view = navigate_to(appliance.collections.cloud_keypairs, 'All')
+    key_pair = view.entities.get_first_entity().data['name']
+    key_pair_obj = appliance.collections.cloud_keypairs.instantiate(key_pair)
+    key_pair_obj.set_ownership("", new_tenant_admin.groups[0].description)
```

Comment:
Why the empty string here?

Reply:
I'm just setting group ownership and not user ownership, so updated the statement so that it's done the right way.